### ISSUE TITLE
Paymetheus must show its current version

### ISSUE THREAD

**Issue by SteffenIvanhoe** [decred/Paymetheus#304]
I again and again ask myself whether I run the latest versíon of Paymetheus. I just cannot find out! It is a very good practice to display the version of the software, either in the window header or at least somewhere in one of the separate looks.

Add the current version (e.g. 1.05) to the window title line!

**Comment by scarmani** [comment]
I agree; version number should be visible in the window header / title, or in a view within the running wallet.  In the meantime, version can be determined by locating the Paymetheus.exe file and looking at the "File Version" property.

**Comment by SteffenIvanhoe** [comment]
Wow, someone has finally read it! After almost 3 months! I meanwhile thought that Paymetheus / Decred was a dead project without any serious developers.
